acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -33,8 +33,8 @@ public String getHostname() {
     return hostname;
   }
 
-  public Duration getPollingInterval() {
-    return POLLING_INTERVAL;
+  public Duration getTimeout() {
```

Comment:
This variable gets passed through to `Web3j` where it names it as `pollingInterval` ...which could be an entirely different idea to a timeout, is it?

Reply:
We are using what for the "--downstream-http-request-timeout" in the EthSignerProcessRunner. That is why I changed it. We should probably change the config option to a different name then ... What do you think?
After changing "--downstream-http-request-timeout" from 500ms to 1s I do not get any timeouts any more on my local machine.

Reply:
The "--downstream-http-request-timeout" is used  in a OkHttpClient.Builder to set the "connectTimeout" and the "readTimeout" ... 
That OkHttpClient is passed into the JsonRpc2_0Web3j constructor.
So I think the name timeout makes sense!

Reply:
I'm referring to the constructor for `JsonRpc2_0Web3j` that names in the input variables as `pollingInterval`.

The point of my question, was whether that is applied in the same way that the EthSigner timeout variable?

Reply:
...as `JsonRpc2_0Web3j` stores the `pollingInterval` as `blockTime` field, which looks to be used for another purpose

Reply:
fixed!